pyanaconda/ui/gui/spokes/datetime_spoke.py

```diff
@@ -892,6 +892,15 @@ def _show_no_ntp_server_warning(self):
 
     def on_ntp_switched(self, switch, *args):
         if switch.get_active():
+            # clean timer
+            if self._start_updating_timer:
+                self._start_updating_timer.cancel()
+                self._start_updating_timer = None
+
+            if self._update_datetime_timer:
+                self._update_datetime_timer.cancel()
+                self._update_datetime_timer = None
+
```

Comment:
Here we don't want to cancel `_update_datetime_timer` (we still want the time in the UI to be updated to the current value) but on the contrary we want to make sure it is running, because it is was cancelled in `_stop_and_maybe_start_time_updating` which planned the `_save_system_time` by` _start_updating_timer`. The `_save_system_time` would re-enable the UI update (`_update_date_timer`) again, but it is not called because the `_start_updating_timer` is cancelled in the patch. So I think the patch could be modified in this way:
```
            # cancel system time update planned by manual setting
            if self._start_updating_timer:
                self._start_updating_timer.cancel()
                self._start_updating_timer = None
            # re-enable UI update because it will not be done by system time update we've
            # just cancelled
            if not self._update_datetime_timer:
                self._update_datetime_timer = Timer()
                self._update_datetime_timer.timeout_sec(1, self._update_datetime)
```